### DISCUSSION TITLE
consulta sobre codigo

### DISCUSSION THREAD

**Discussion by Juampi44** [INGCOM-UNRN-P1/cursada-2023#62] [Preguntas y respuestas] [answered]
En el tp1 tengo corregido uno de los ejercicios en el que el codigo estaba mal, puedo enviar por aca el codigo modificado para que me digan si esta bien o no es lo que pide?

**Comment by martinvilu** [comment]
Cargalo en el repositorio y comparti el enlace por acá

**Comment by Juampi44** [comment, marked as answer]
https://github.com/Juampi44/friendly-potato es el archivo de pruebass.c, asi seria o a que se refiere con lazos?

**Reply by martinvilu** [reply]
¿Pero por qué en otro repositorio?

El contexto ya está dado en la entrega corregida, de esta forma se pierde en el camino… y es necesario ir a buscarlo.

**Reply by martinvilu** [reply]
corregilo y entregalo

**Reply by martinvilu** [reply]
se ve razonable
